src/components/Dropdown.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <!-- Copied and modified from Vuetensils to match the existing bulma implementation -->
+  <!-- https://github.com/AustinGil/vuetensils/blob/production/src/components/VDropdown/VDropdown.vue -->
+  <div
+    class="navbar-item has-dropdown is-hoverable"
+    @mouseenter="isHovered = true"
+    @mouseleave="isHovered = false"
+    @focus="isFocused = true"
+    @blur="isFocused = false"
+    @focusout="onFocusout"
+  >
+    <div
+      :aria-expanded="!!isHovered || !!isFocused"
```

Comment:
Can isHovered ever be not boolean? Do we really need the !! in front of it?

Reply:
I think you're right 👍

Reply:
Done in 25b141e